tests/core/test_kontainer_engine_config.py

```diff
@@ -0,0 +1,29 @@
+from .common import random_str
+
+
+def test_gke_config_appears_correctly(admin_mc, remove_resource):
+    cluster = admin_mc.client.create_cluster(
+        name=random_str(), googleKubernetesEngineConfig={
+            "credentials": "bad credentials"
+        })
+    remove_resource(cluster)
+
+    # test that a cluster returned from a POST has the correct config
+    assert cluster.googleKubernetesEngineConfig is not None
```

Comment:
After (or instead of of the None check) check that the `credentials` field is actually set. An empty dict will pass this test.

Reply:
Done